internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -107,7 +108,6 @@ func resourceLogAnalyticsWorkspace() *pluginsdk.Resource {
 				Optional: true,
 				Computed: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					string(workspaces.WorkspaceSkuNameEnumFree),
```

Comment:
Users who've previously provisioned a Resource using the `Free` SKU will be able to continue using it until they migrate to a different SKU - so removing this SKU would break support for those users?

Reply:
Hi @tombuildsstuff - It's removed in the next major release. Do you think we should keep it in the validation func in the next major release?

Reply:
@ms-henglu, how long has the free option no longer been valid? i think if it JUST changed maybe we want to keep the validation around in schema and maybe add a customize diff to prevent new ones from being created but if its been gone for a year then maybe it time to remove it?

Reply:
Hi @katbyte - The `Free` SKU was possible only until July 1, 2022. It's more than 2 years, I guess we could remove it.

Reply:
that has been a while, i think its fine to remove then